internal/generator/vector/conf/complex.toml

```diff
@@ -4,6 +4,11 @@ data_dir = "/var/lib/vector/openshift-logging/my-forwarder"
 [api]
 enabled = true
 
+# Load sensitive data from secret mount with script: /usr/bin/read_secret_data.sh
+[secret.from_secret_mount]
+type = "exec"
+command = ["/usr/bin/read_secret_data.sh"]
```

Comment:
do we need to set permissions on this when mounting the volume?

Reply:
you mean run permissions?